packages/turbo-codemod/src/commands/migrate/steps/getLatestVersion.ts

```diff
@@ -12,9 +12,12 @@ interface PackageDetailsResponse {
 }
 
 async function getPackageDetails({ packageName }: { packageName: string }) {
+  const registry =
+    process.env.npm_config_registry?.replace(/\/$/, "") || DEFAULT_REGISTRY;
```

Comment:
Can you explain the reason for this `.replace` here? grep.app [doesn't show me any prior art](https://grep.app/search?q=process.env.npm_config_registry%3F.replace%28/%5C/%24/%2C%20%22%22%29) for this, though [without the regex](https://grep.app/search?q=process.env.npm_config_registry) does. I'm figuring there's something I'm missing, just can't tell what it is.

Reply:
The reason is to convert `https://registry.npmjs.org/` to `https://registry.npmjs.org`

Happy to remove it though.

Reply:
Of course!